extra-3

```diff
@@ -0,0 +1 @@
+
```

Comment:
Just remove the file. As in, delete it, then commit your changes (deleting a file from your repo is a change), then squash commits properly.